### PR TITLE
Stdet config

### PR DESCRIPTION
**Pull request by kennymckormick**

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -396,6 +396,234 @@ which is convenient to conduct various experiments.
...
+        assigner=dict(  # Dict for assigner
+            type='MaxIoUAssignerAVA', # Name of the assigner
+            pos_iou_thr=0.9,  # IoU threshold for positive examples, > pos_iou_thr -> positive
+            neg_iou_thr=0.9,  # IoU threshold for negative examples, < neg_iou_thr -> negative
+            min_pos_iou=0.9), # Minimum acceptable IoU for positive examples
+        sampler=dict( # Dict for sample
+            type='RandomSampler', # Name of the sampler
+            num=32, # Batch Size of the sampler
+            pos_fraction=1, # Positive bbox fraction of the sampler
+            neg_pos_ub=-1,  # Positive bbox fraction of the sampler
```
Comment (innerlee) [inline docs/config.md:451]: Check comment of this line

File: docs/config.md [hunk 2]
```diff
@@ -396,6 +396,234 @@ which is convenient to conduct various experiments.
...
+            add_gt_as_proposals=True), # Upper bound of the ratio of num negative to num positive
```
Comment (innerlee) [inline docs/config.md:452]: check this line also

File: docs/config.md [hunk 3]
```diff
@@ -396,6 +396,234 @@ which is convenient to conduct various experiments.
...
+
+```python
+# model setting
+model = dict( # Config of the model
+    type='FastRCNN',  # Type of the detector
+    backbone=dict(  # Dict for backbone
+        type='ResNet3dSlowOnly',  # Name of the backbone
+        depth=50, # Depth of ResNet model
+        pretrained=None,   # The url/site of the pretrained model
+        pretrained2d=False, # If the pretrained model is 2D
```
Comment (innerlee) [inline docs/config.md:418]: Unrelated to this pr.

`pretrained2d` -> `is_pretrained2d` might be clearer

**Comment by innerlee** [comment]
Looks good

File: docs/config.md [hunk 4]
```diff
@@ -396,6 +396,234 @@ which is convenient to conduct various experiments.
...
+        data_prefix=data_root))
+data['test'] = data['val']    # Set test_dataset as val_dataset
+# optimizer
+optimizer = dict(
+    # Config used to build optimizer, support (1). All the optimizers in PyTorch
+    # whose arguments are also the same as those in PyTorch. (2). Custom optimizers
+    # which are builed on `constructor`, referring to "tutorials/new_modules.md"
+    # for implementation.
+    type='SGD',  # Type of optimizer, refer to https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/optimizer/default_constructor.py#L13 for more details
+    lr=0.2,  # Learning rate, see detail usages of the parameters in the documentaion of PyTorch
```
Comment (congee524) [inline docs/config.md:590]: how many gpus were used when setting lr = 0.2?

**Comment by congee524** [comment]
Looks good to me.

File: docs/config.md (hunk 3 above)
Comment (kennymckormick) [reply docs/config.md:418]: I think this flag can even be removed since when the checkpoint is loaded, you know if it is 2d or 3d (by looking at conv.weight shape, etc.).

File: docs/config.md (hunk 4 above)
Comment (kennymckormick) [reply docs/config.md:590]: added

File: docs/config.md (hunk 3 above)
Comment (innerlee) [reply docs/config.md:418]: yeah

**Comment by innerlee** [comment]
Forgot to push?

**Review by innerlee — APPROVED** [review]
